My task ID: 6
Number of Tasks: 96

TRAINING DATA

Accepting Grid:
SET_combinations | Number of Trials | Status
GGG | 100 | 
GPR | 100 | 
GRP | 100 | 
PGR | 100 | 
PPP | 100 | 
PRG | 100 | 
RGP | 100 | 
RPG | 100 | 
RRR | 100 | 

Rejecting Grid:
SET_combinations | Number of Trials | Status
GGP | 50 | 
GGR | 50 | 
GPG | 50 | 
GPP | 50 | 
GRG | 50 | 
GRR | 50 | 
PGG | 50 | 
PGP | 50 | 
PPG | 50 | 
PPR | 50 | 
PRP | 50 | 
PRR | 50 | 
RGG | 50 | 
RGR | 50 | 
RPP | 50 | 
RPR | 50 | 
RRG | 50 | 
RRP | 50 | 

----------

TESTING DATA

Accepting Grid:
SET_combinations | Number of Trials | Status
GGG | 5 | 
GPR | 5 | 
GRP | 5 | 
PGR | 5 | 
PPP | 5 | 
PRG | 5 | 
RGP | 5 | 
RPG | 5 | 
RRR | 5 | 

Rejecting Grid:
SET_combinations | Number of Trials | Status
GGP | 5 | 
GGR | 5 | 
GPG | 5 | 
GPP | 5 | 
GRG | 5 | 
GRR | 5 | 
PGG | 5 | 
PGP | 5 | 
PPG | 5 | 
PPR | 5 | 
PRP | 5 | 
PRR | 5 | 
RGG | 5 | 
RGR | 5 | 
RPP | 5 | 
RPR | 5 | 
RRG | 5 | 
RRP | 5 | 

----------

GROK DATA

Accepting Grid:
SET_combinations | Number of Trials | Status

Rejecting Grid:
SET_combinations | Number of Trials | Status

----------

CORRUPT DATA

Accepting Grid:
SET_combinations | Number of Trials | Status

Rejecting Grid:
SET_combinations | Number of Trials | Status


Metrics after epoch 50:
train_loss: 0.9489211440086365
train_accuracy: 0.0318080373108387
test_loss: 1.0352789163589478
test_accuracy: 0.007407407276332378
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 100:
train_loss: 0.0034104587975889444
train_accuracy: 0.9988839030265808
test_loss: 0.0035418043844401836
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 150:
train_loss: 0.0012668954441323876
train_accuracy: 1.0
test_loss: 0.0015936846612021327
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 200:
train_loss: 0.0006293190526776016
train_accuracy: 1.0
test_loss: 0.0006332971388474107
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 250:
train_loss: 0.0003301432298030704
train_accuracy: 1.0
test_loss: 0.0003765046421904117
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 300:
train_loss: 0.00028276367811486125
train_accuracy: 1.0
test_loss: 0.00028031051624566317
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 350:
train_loss: 0.0001308773789787665
train_accuracy: 1.0
test_loss: 0.0001265419414266944
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 400:
train_loss: 7.00034579494968e-05
train_accuracy: 1.0
test_loss: 7.420644396916032e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 450:
train_loss: 4.321548476582393e-05
train_accuracy: 1.0
test_loss: 4.73248801426962e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 500:
train_loss: 0.0009964490309357643
train_accuracy: 0.9977678656578064
test_loss: 0.00030212473939172924
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 550:
train_loss: 3.1783773010829464e-05
train_accuracy: 1.0
test_loss: 3.0606035579694435e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 600:
train_loss: 1.8372580598224886e-05
train_accuracy: 1.0
test_loss: 2.0535049770842306e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 650:
train_loss: 1.3741373550146818e-05
train_accuracy: 1.0
test_loss: 1.5564799468847923e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 700:
train_loss: 1.0056887731479947e-05
train_accuracy: 1.0
test_loss: 1.1220685337320901e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 750:
train_loss: 7.806825124134775e-06
train_accuracy: 1.0
test_loss: 1.2484871149354149e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 800:
train_loss: 1.82454696187051e-05
train_accuracy: 1.0
test_loss: 1.741796950227581e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 850:
train_loss: 1.290485215577064e-05
train_accuracy: 1.0
test_loss: 9.005824722407851e-06
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 900:
train_loss: 1.1067894774896558e-05
train_accuracy: 1.0
test_loss: 1.6051586499088444e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 950:
train_loss: 6.10838378634071e-06
train_accuracy: 1.0
test_loss: 7.107131750672124e-06
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 1000:
train_loss: 4.871832970820833e-06
train_accuracy: 1.0
test_loss: 3.728927822521655e-06
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


LLSUB_RANK=6: 2023-09-20 11:49:15.764905: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
LLSUB_RANK=6: 2023-09-20 11:49:19.475100: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer.so.7'; dlerror: libnvinfer.so.7: cannot open shared object file: No such file or directory
LLSUB_RANK=6: 2023-09-20 11:49:19.475178: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer_plugin.so.7'; dlerror: libnvinfer_plugin.so.7: cannot open shared object file: No such file or directory
LLSUB_RANK=6: 2023-09-20 11:49:19.475188: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.
LLSUB_RANK=6: /state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/__init__.py:98: UserWarning: unable to load libtensorflow_io_plugins.so: unable to open file: libtensorflow_io_plugins.so, from paths: ['/state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/libtensorflow_io_plugins.so']
LLSUB_RANK=6: caused by: ['/state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/libtensorflow_io_plugins.so: undefined symbol: _ZN3tsl6StatusC1EN10tensorflow5error4CodeESt17basic_string_viewIcSt11char_traitsIcEENS_18SourceLocationImplE']
LLSUB_RANK=6:   warnings.warn(f"unable to load libtensorflow_io_plugins.so: {e}")
LLSUB_RANK=6: /state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/__init__.py:104: UserWarning: file system plugins are not loaded: unable to open file: libtensorflow_io.so, from paths: ['/state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/libtensorflow_io.so']
LLSUB_RANK=6: caused by: ['/state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/libtensorflow_io.so: undefined symbol: _ZTVN3tsl13GcsFileSystemE']
LLSUB_RANK=6:   warnings.warn(f"file system plugins are not loaded: {e}")
LLSUB_RANK=6: No GPU/TPU found, falling back to CPU. (Set TF_CPP_MIN_LOG_LEVEL=0 and rerun for more info.)
LLSUB_RANK=6: 2023-09-20 11:50:27.421854: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
LLSUB_RANK=6: 2023-09-20 11:50:27.421881: W tensorflow/compiler/xla/stream_executor/cuda/cuda_driver.cc:265] failed call to cuInit: UNKNOWN ERROR (303)
